src/qComponents/QSlider/src/types.ts

```diff
@@ -7,6 +7,8 @@ export type QSliderPropModelValue<T extends ModelValue = ModelValue> = T;
 
 type SlotData = Optional<Record<string | number, unknown>>;
 
+export type QSliderPropTooltipMode = 'none' | 'hover' | 'always';
```

Comment:
вроде бы null достаточно вместо none

```suggestion
export type QSliderPropTooltipMode = Nullable<'hover' | 'always'>;
```